app.json

```diff
@@ -23,6 +23,9 @@
     "assetBundlePatterns": [
       "**/*"
     ],
+    "android": {
+      "package": "com.ritalin.Paylist"
```

Comment:
@offler9 it can be `com.ritalin.paylist`
here some reference:
https://stackoverflow.com/questions/6273892/android-package-name-convention